console/src/components/Services/Actions/Relationships/utils.js

```diff
@@ -90,3 +90,15 @@ export const removeTypeRelationship = (types, typename, relName) => {
     return t;
   });
 };
+
+export const validateRelTypename = (types, typename, relname) => {
+  for (let i = types.length - 1; i >= 0; i--) {
+    const type = types[i];
+    if (type.kind === 'object' && type.name === typename) {
+      if ((type.relationships || []).find(r => r.name === relname)) {
```

Comment:
Since you just want a boolean and don't care about the result of `find` what about using `some` here?